pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -190,7 +190,12 @@
     public static final String CONFIG_OF_QUERY_EXECUTOR_CLASS = "pinot.server.query.executor.class";
     public static final String CONFIG_OF_REQUEST_HANDLER_FACTORY_CLASS = "pinot.server.requestHandlerFactory.class";
     public static final String CONFIG_OF_NETTY_PORT = "pinot.server.netty.port";
+    public static final String CONFIG_OF_ENABLE_GRPC_SERVER = "pinot.server.grpc.enable";
+    public static final boolean DEFAULT_ENABLE_GRPC_SERVER = false;
+    public static final String CONFIG_OF_GRPC_PORT = "pinot.server.grpc.port";
```

Comment:
I think we need to expose this port through helix eventually.

Reply:
Added the port info into the Helix `InstanceConfig` under `grpcPort`